src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,60 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
+                .joinToString { it.text() }
+        }
+        val GroupName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Group) + td:nth-child(2) > a")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Group) + td:nth-child(2) > a")
+                .joinToString { it.text() }
+        }
+        val CharacterName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Character) + td:nth-child(2) > a")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Character) + td:nth-child(2) > a")
+                .joinToString { it.text() }
+        }
+        val SeriesName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a")
+                .joinToString { it.text() }
+        }
+        val SeriesParser = if ("Manhwa" in SeriesName) {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr:nth-child(5) > td:nth-child(2)").text()
+        } else {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.parodies > td:nth-child(2) > a").text()
+        }
+        val AlternativeTitle = when {
+            document.select("section.metadata > h1.title > span.alter")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> document.select("section.metadata > h1.title > span.alter")
+                .joinToString { it.text() }
+        }
         val manga = SManga.create()
         manga.description = when {
             document.select("section.metadata > div.pb-2 > p:nth-child(1)")
-                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque"
-            else -> document.select("section.metadata > div.pb-2 > p:nth-child(1)").first()!!.text()
+                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque\n\nJudul Alternatif : $AlternativeTitle"
+            else -> (document.select("section.metadata > div.pb-2 > p:nth-child(1)").first()!!.text() + "\n\nJudul Alternatif : $AlternativeTitle")
         }
         val genres = mutableListOf<String>()
         infoElement.select("div.tags > a").forEach { element ->
             val genre = element.text()
             genres.add(genre)
         }
-        manga.author =
-            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Author) + td:nth-child(2) > a")
-                .joinToString { it.text() }
+        manga.author = "Author : $AuthorName\nGroup  : $GroupName"
```

Comment:
Group, or 'circle' belongs in the description. Or if it is the scanlation group it should be on the chapter.

Reply:
It Group for Doujin maker, so i'll move it to the desc with Chara and Series/Serilization along with Alternative Title